experiments/configs/efficientnet_b0_classification_cifar100.yml

```diff
@@ -27,20 +29,20 @@ dataset: {
     }]
   },
   eval: {
-    dataset: CIFAR100,
+    name: CIFAR100,
     args: {
       root: external/datasets,
       train: False,
       download: True
     }
   },
-  dataloader: {
-    dataloader: DataLoader,
-    args: {
-      num_workers: 0,
-      batch_size: 256,
-      shuffle: True,
-    },
+}
+dataloader: {
+  module: PytorchDataLoader,
```

Comment:
this name follows #38 right? not sure if this is the best name

Reply:
I mean we are using pytorch, so I think it is kind of obvious that default loader is from torch

Reply:
Any suggestion?

Reply:
How about 'DefaultDataLoader', do you think its better? @alifahrri

Reply:
DefaultLoader?

Reply:
it think it's better to follow original name, `DataLoader`

Reply:
btw, if we stick with `PytorchDataLoader`, this may implies that there is `TFDataLoader`, `MxNetDataLoader`, etc

Reply:
before this, @alphinside proposed to use `DeafultLoader`. how about that?

Reply:
no problem, as long as no `Pytorch` prefix

Reply:
btw I think it doesn't have to be resolved in this PR since this is somewhat out of scope

Reply:
but yeah if everyone agrees then there is no problem

Reply:
ok for now lets go with this